src/system_data.jl

```diff
@@ -935,8 +955,13 @@ end
 _get_system_basename(system_file) = splitext(basename(system_file))[1]
 _get_secondary_basename(system_basename, name) = system_basename * "_" * name
 
-add_supplemental_attribute!(data::SystemData, component, info; kwargs...) =
-    add_supplemental_attribute!(data.attributes, component, info; kwargs...)
+function add_supplemental_attribute!(data::SystemData, component, info; kwargs...)
```

Comment:
Seems a left over use of `info` 
```suggestion
function add_supplemental_attribute!(data::SystemData, component, attribute; kwargs...)
```